en/developing-web-services.html

```diff
@@ -80,10 +80,9 @@ <h2 id="accessing-the-http-request">Accessing the HTTP request</h2>
 the <code>com.yahoo.jdisc.Request</code> can be cast to an instance
 of <code>com.yahoo.jdisc.http.HttpRequest</code> when the request is over HTTP.
 Web service handlers should usually extend
-<code>com.yahoo.container.jdisc.ThreadedHttpRequestHandler</code>
-(or the <code>com.yahoo.container.jdisc.LoggingRequestHandler</code> subclass
-which also does access logging),
-which provides to subclasses a <code>com.yahoo.container.jdisc.HttpRequest</code>
+<code><a href="https://javadoc.io/doc/com.yahoo.vespa/container-core/latest/com/yahoo/container/jdisc/ThreadedHttpRequestHandler.html">
+    ThreadedHttpRequestHandler</a></code> which provides to subclasses a
```

Comment:
I did not understand that sentence

Reply:
Agree, I just used what was there earlier. Will rewrite.